grails-app/conf/Config.groovy

```diff
@@ -207,7 +207,8 @@ log4j = {
 
     root {
         error "tomcatLog"
-        warn "tomcatLog", "stdout"
+        warn "tomcatLog"
```

Comment:
isn't tomcatLog now duplicated?

Reply:
It was duplicated before also because if you define it in warn level then error level isn't needed because of hierarchy. But it doesn't affect anything